### PR TITLE
Alter Table court_location 

### PR DESCRIPTION
**Pull request by jerryfreshworks**
- new migration to update exists court_location, 
1. removing `location_number`
2. add `location_code`, `lat`, `lng`
3. Integrate with justice location api
4. Create e2e tests
5. configure cronjob 
6. Integrate mapbox
7. Integrate active list

- change booking.entity filed

- frontend: removing duplicate langs, fetching location data and populating dropdown.

### CODE REVIEW

File: api/package.json [hunk 1]
```diff
@@ -28,11 +28,13 @@
     "doc:generate": "npm-run-all typedoc test:cov"
   },
   "dependencies": {
+    "@mapbox/mapbox-sdk": "^0.12.1",
```
Comment (jerryfreshworks) [inline api/package.json:31]: maybe remove the `^`

File: api/src/migrations/1618521340231-UpdateLocationColumns.ts [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+    // clean table
+    await queryRunner.query(`DELETE FROM ${Table}`);
+
+    // add column location_number
+    await queryRunner.query(`ALTER TABLE ${Table} 
+     ADD COLUMN location_number int;`);
+
+    //  delete location_code
+    await queryRunner.query(`ALTER TABLE ${Table} DROP COLUMN "location_code"`);
+
```
Comment (jerryfreshworks) [inline api/src/migrations/1618521340231-UpdateLocationColumns.ts:34]: you may need a revert of `await queryRunner.query(`ALTER TABLE ${Table} ADD COLUMN "location_short_desc" VARCHAR NOT NULL`);`

Comment (fwpushan) [reply api/src/migrations/1618521340231-UpdateLocationColumns.ts:34]: Good I will do that

File: api/package.json (hunk 1 above)
Comment (fwpushan) [reply api/package.json:31]: 👍

**Review by fwpushan — APPROVED** [review]
from @jerryfreshworks

**Pull request merged**